config.py

```diff
@@ -0,0 +1,5 @@
+DEBUG = True
```

Comment:
It should be false in production, yes, but (and it's probably a bit confusing) `config.py` is only used by `run.py`, which contains a barebones Flask app that serves just this blueprint. For testing purposes, presumably. `config.py` isn't used for anything else.

This is a pattern from `flask-govdelivery` and `flask-pdfreactor`.